### DISCUSSION TITLE
Compose placeholders are hidden and content entered into a new document isnt visible

### DISCUSSION THREAD

**Discussion by lakshmibhargavaBo-Kore** [outline/outline#5567] [Self Hosting] [answered]
**To Reproduce**
1. Go to any collection
2. Create a new document and add the title to it
3. Click on compose area and try to add some content to it.
4. you wouldn't see any of the text you have added
5. Click on any other document and come back to the document you created earlier
6. You would be able to see the text you entered earlier

**Expected behavior**
you should be able to the entered text into a new document as soon as you entered the content

**Screenshots**
If applicable, add screenshots or videos to help explain your problem.

**Outline (please complete the following information):**
- Install: self-hosted
- Version: 0.70.2

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Chrome
 - Version: 114.0.5735.199

**Comment by tommoor** [comment, marked as answer]
This sounds like websockets are not connecting which would be configuration issue with whatever proxy you're using, please search discussions as this has come up many times.

**Reply by lakshmibhargavaBo-Kore** [reply]
https://github.com/outline/outline/discussions/3677
This helped
